modules/stackrox-kubernetes-security-platform-support.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support-stackrox.adoc
+:_module-type: CONCEPT
+[id="stackrox-kubernetes-security-platform-support_{context}"]
+= StackRox Kubernetes Security Platform support
+
+[role="_abstract"]
+StackRox supports the StackRox Kubernetes Security Platform versions for up to 6 months after its release, which corresponds to the previous 9 released versions. StackRox will make reasonable efforts and assist you in supporting some older versions. However, the support team may request you to upgrade to a newer released version of the StackRox Kubernetes Security Platform for full support.
+
+[NOTE]
+====
+For the StackRox Kubernetes Security Platform, StackRox supports:
+
+* The latest released version (which we’ll refer to as N).
+* 9 earlier versions.
+
+The support window of the StackRox Kubernetes Security Platform versions is known as N-9: (N (Latest release) - 9 (earlier versions)).
```

Comment:
It's okay to use the numeral "9" in these references. The outer parentheses should be square brackets, so:
[N (Latest release) - 9 (earlier versions)]. However, I think we can reduce this confusion further by the addition of "where". For example:

> The support window of the StackRox Kubernetes Security Platform versions is known as N-9 
> where: 
> N (latest release) - 9 (earlier versions)

ISG gives several examples of using the word _where_ with a lowercase _w_ to introduce an explanation.

Or you could just use it in a running sentence: 

> The support window of the StackRox Kubernetes Security Platform versions is known as N-9, where: N (latest release) - 9 (earlier versions).

(Also, note the lowercase _l_ in _latest_.)